src/reducers/auth.ts

```diff
@@ -0,0 +1,36 @@
+import * as Constants from 'constants/auth'
+import { AuthState } from 'contexts/auth'
+
+/* eslint-disable camelcase */
+type AuthTypePayload = {
+  attributes: {
+    access_token: string
+    refresh_token: string
+    token_type: string
+  }
+}
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
this line is not needed anymore I think.

Reply:
Good catch. Fixed in 836a7b0dedd5b56cee09f4faeb45c518782b8688